rate.go

```diff
@@ -15,38 +15,46 @@ type rediser interface {
 	Pipelined(func(pipe *redis.Pipeline) error) ([]redis.Cmder, error)
 }
 
+//Limiter type
 type Limiter struct {
 	fallbackLimiter *timerate.Limiter
 	redis           rediser
 }
 
-// A Limiter controls how frequently events are allowed to happen. It uses
-// the redis to store data and fallbacks to the fallbackLimiter
-// when Redis Server is not available.
+// NewLimiter creates a limiter that controls how frequently events
+// are allowed to happen. It uses redis to store data and fallbacks
+// to the fallbackLimiter when Redis Server is not available.
```

Comment:
I know you did not change, but as remindment to myself this comment should be on `struct Limiter`